app/utils/searchUtils.js

```diff
@@ -18,9 +18,10 @@ function getFetchParamsFromFilters(filters) {
       filters.duration
     ),
     { purpose: filters.purpose === 'all' ? '' : filters.purpose },
-    { page: filters.page || 1 }
+    { page: filters.page || 1 },
+    { ...(filters.orderBy ? { order_by: filters.orderBy } : {}) },
+    { ...(filters.available_between ? { available_between: filters.available_between } : {}) }
```

Comment:
What is this `available_between` line for, is it leftovers from the `open_now` sort option?

Reply:
Yes, it was missed to remove. I have removed it in the recent commit.